public/docs/ts/latest/guide/architecture.jade

```diff
@@ -300,21 +286,21 @@ figure
 .l-main-section
 :marked
   ## Data binding
-  Without a framework, we would be responsible for pushing data values into the HTML controls and turning user responses
+  Without a framework, you would be responsible for pushing data values into the HTML controls and turning user responses
   into actions and value updates. Writing such push/pull logic by hand is tedious, error-prone, and a nightmare to
   read as any experienced jQuery programmer can attest.
 figure
   img(src="/resources/images/devguide/architecture/databinding.png" alt="Data Binding" style="width:220px; float:left; margin-left:-40px;margin-right:20px" )
 :marked
   Angular supports **data binding**,
   a mechanism for coordinating parts of a template with parts of a component.
-  We add binding markup to the template HTML to tell Angular how to connect both sides.
+  Add binding markup to the template HTML to tell Angular how to connect both sides.
 
   There are four forms of data binding syntax. Each form has a direction &mdash; to the DOM, from the DOM, or in both directions &mdash;
```

Comment:
The "There are" makes me cringe, but I can't think of anything better. Still, we could tie this better to the figure. How about something like this:

As the diagram shows, there are four forms of data binding syntax... both directions.